cmd/personality.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2014-2024 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"strings"
+)
+
+// ProgramName represents the name of the application binary.
+var ProgramName string = "pebble"
+
+// DisplayName represents the user-facing name of the application.
+var DisplayName string = "Pebble"
+
+// DefaultDir is the Pebble directory used if $PEBBLE is not set.
+var DefaultDir string = "/var/lib/pebble/default"
+
+func ApplyPersonality(s string) string {
```

Comment:
Let's be conservative and keep this in internals/cli for now, so we're not locked into it in a public package.

Reply:
Done :)